engines/director/detection_paths.h

```diff
@@ -133,8 +142,48 @@ const char *directoryGlobs[] = {
 	"dicfiles",						// Time Blazers series
 	"docs",							// Enhanced CDs
 	"anim",							// 1 vs 100 (Mindscape)
-	"Lingo Expo",					// D4 sample movies
+	"lingo expo",					// D4 sample movies
 	"main",							// Meet MediaBand
+	"toinstall",					// Jönssonligan series
```

Comment:
It would be best to sort these alphabetically as well, so we could spot the duplicates easily.

Reply:
The reason that these are not sorted alphabetically is that the games are grouped together. If we change the sorting, we'd need to label every line to know which game(s) need(s) it.